package/network/utils/wireguard-tools/files/wireguard.sh

```diff
@@ -42,6 +44,11 @@ proto_wireguard_setup_peer() {
 	config_get endpoint_port "${peer_config}" "endpoint_port"
 	config_get persistent_keepalive "${peer_config}" "persistent_keepalive"
 
+	if [ "${disabled}" -eq 1 ]; then
```

Comment:
I don't really see the point of using {} brackets without adjacent characters, but since it's used elsewhere as well I can live with it ...